src/Sylius/Behat/Context/Ui/Admin/NotificationContext.php

```diff
@@ -63,6 +63,18 @@ public function iShouldBeNotifiedThatItHasBeenSuccessfullyDeleted(): void
         );
     }
 
+    /**
+     * @Then I should be notified that the removal operation has started successfully
+     */
+    public function iShouldBeNotifiedThatTheRemovalOperationHasStartedSuccessfully(): void
+    {
+        $this->testHelper->waitUntilNotificationPopups(
+            $this->notificationChecker,
+            NotificationType::success(),
+            'has been successfully deleted.',
```

Comment:
How come this passes, when the message is supposed to be:

> Removing of catalog promotion has been ordered. This process can take a while depending on the number of products.

?

Reply:
Good question, I checked it and it looks like we can put whatever we want and the test passes. I'll try to figure out what's going on

Reply:
[This PR](https://github.com/Sylius/Sylius/pull/14301) should fix it